tests/unit/test_vcs.py

```diff
@@ -366,3 +376,50 @@ def test_subversion__get_url_rev_options():
 def test_get_git_version():
     git_version = Git().get_git_version()
     assert git_version >= parse_version('1.0.0')
+
+
+@pytest.mark.svn
+def test_subversion__get_vcs_version():
+    """
+    Test Subversion.get_vcs_version() against local ``svn``.
+    """
+    version = Subversion().get_vcs_version()
+    assert len(version) == 3
+    assert version[0] >= 1
+
+
+@pytest.mark.parametrize('svn_output, expected_version', [
+    ('svn, version 1.10.3 (r1842928)\n'
+     '   compiled Feb 25 2019, 14:20:39 on x86_64-apple-darwin17.0.0',
+     (1, 10, 3)),
+    ('svn, version 1.9.7 (r1800392)\n'
+     '   compiled Mar 28 2018, 08:49:13 on x86_64-pc-linux-gnu',
+     (1, 9, 7)),
+])
+@patch('pip._internal.vcs.subversion.Subversion.run_command')
+def test_subversion__get_vcs_version_patched(mock_run_command, svn_output,
+                                             expected_version):
+    """
+    Test Subversion.get_vcs_version() against patched output.
+    """
+    mock_run_command.return_value = svn_output
+    version = Subversion().get_vcs_version()
+    assert version == expected_version
+
+
+@pytest.mark.parametrize('svn_output', [
+    ('svn, version 1.10.z (r1842928)\n'
+     '   compiled Feb 25 2019, 14:20:39 on x86_64-apple-darwin17.0.0'),
+    '',
+    'svn, version . .'
```

Comment:
Another failure case to test is `run_command()` raising `BadCommand`. You can test this by setting `side_effect` on the mock object.